### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by JTtheGeek** [Deadrank/DeadRank-DU#19] [General]
Just wanted to say thanks for your outstanding work on this.  PVP in DU already has a huge barrier to entry, and this HUD makes things a lot nicer.    If I would add anything it would be a way to switch to a mode that moves to auto intercept and specified range.  Possibly having the mouse wheel control the ideal range, and using target trajectory estimation to keep the target in rage.    A possible other mode might be dump and run strat where it get's within range, waits until the guns are nearing empty and gets out of range while reloading, rinse and repeat.   Thanks again though, the whole platform works, well, your code is very readable and well organized!

**Comment by Deadrank** [comment]
Hey! Glad you are enjoying and using the HUD. My whole goal with it was lowering the barrier to entry into DU PvP. Unfortunately not enough data is given through the radar API to do directional matching of an enemy target. The only details that NQ allows us to pull from the radar that are helpful for what you suggest are the distance and speed of the target. While it would be potentially possible to add in a sort of engine control system that would help keep you at an optimal range, it is counter production to the current meta (freight engines) because it would be constantly turning the engines off and then back on and when using freight engines, they need to stay at 100% throttle throughout the flight due to their long warmup times.
The 2nd reason I probably won't be adding any type of combat following scripting is that being in control of your ship and actually piloting it yourself are what make someone better at PvP in DU than another person given all else is equal. This allows pilots to show off their skills :)
